app/src/main/java/com/example/diceroller/RolledDiceViewModel.kt

```diff
@@ -1,65 +0,0 @@
-package com.example.diceroller
-
-import androidx.lifecycle.ViewModel
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.asStateFlow
-import java.text.DateFormat
-import kotlin.random.Random
-
-class RolledDiceViewModel : ViewModel() {
-    private val _currentDice: MutableStateFlow<DiceType> = MutableStateFlow(DiceType.D6)
-    val currentDice = _currentDice.asStateFlow()
-
-    private val _listOfDices: MutableStateFlow<List<Dice>> = MutableStateFlow(listOf<Dice>())
-    val listOfDices = _listOfDices.asStateFlow()
```

Comment:
💯  Very smart to use StateFlow! It's my preferred way of exposing state to the view. It does add some complication to your code though. Because Flow is a new API and built with coroutines in mind you have to add a lot of additional handling. 

Flow and Coroutines is all about managing asynchronous operations. Because of that you fall into the world of Threads. Notice how you needed to add `lifecycleScope.launch{}` to your View? This is because Flow assumes it can be accessed from any thread.